### PR TITLE
[it] Align the concepts/overview/components page to it's English version

### PR DESCRIPTION
**Pull request by Fale**
Align the Italian translation of [concepts/overview/components](https://kubernetes.io/docs/concepts/overview/components/) to the English version

### CODE REVIEW

File: content/it/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -46,12 +42,16 @@ Vedi [creare un cluster in High-Availability](/docs/admin/high-availability/) pe
...
 
-Alcuni esempi di controller gestiti dal kube-controller-manager sono:
+Ci sono molti tipi differenti di controller. Alcuni esempi sono:
+
+  * Node controller: Responsabile del monitoraggio e della reazione quando i nodi del cluster diventano irraggiungibili.
+  * Job controller: Monitora gli oggetti di tipo Job, i quali rappresentano
+    delle operazioni una tantum, e crea i Pod necessari per la loro esecuzione.
+  * EndpointSlice controller: Popola gli oggetti EndpointSlice (che servono per
+    collegare i Service e i Pod).
+  * Service Account controllers: Crea i ServiceAccount di default per i nuovi namespace.
```
Comment (sftim) [inline content/it/docs/concepts/overview/components.md:52]:
```suggestion
  * Controllori ServiceAccount: Crea i ServiceAccount di default per i nuovi namespace.
```

File: content/it/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -46,12 +42,16 @@ Vedi [creare un cluster in High-Availability](/docs/admin/high-availability/) pe
...
 {{< glossary_definition term_id="kube-controller-manager" length="all" >}}
 
-Alcuni esempi di controller gestiti dal kube-controller-manager sono:
+Ci sono molti tipi differenti di controller. Alcuni esempi sono:
+
+  * Node controller: Responsabile del monitoraggio e della reazione quando i nodi del cluster diventano irraggiungibili.
+  * Job controller: Monitora gli oggetti di tipo Job, i quali rappresentano
+    delle operazioni una tantum, e crea i Pod necessari per la loro esecuzione.
+  * EndpointSlice controller: Popola gli oggetti EndpointSlice (che servono per
+    collegare i Service e i Pod).
```
Comment (sftim) [inline content/it/docs/concepts/overview/components.md:51]: I'd use “controllore”, as in “controllore logico programmabile”.

File: content/it/docs/concepts/overview/components.md [hunk 3]
```diff
@@ -46,12 +42,16 @@ Vedi [creare un cluster in High-Availability](/docs/admin/high-availability/) pe
 
 {{< glossary_definition term_id="kube-controller-manager" length="all" >}}
 
-Alcuni esempi di controller gestiti dal kube-controller-manager sono:
+Ci sono molti tipi differenti di controller. Alcuni esempi sono:
```
Comment (sftim) [inline content/it/docs/concepts/overview/components.md:45]:
```suggestion
Ci sono molti tipi differenti di _controller_ (controllore). Alcuni esempi sono:
```

File: content/it/docs/concepts/overview/components.md [hunk 4]
```diff
@@ -16,11 +16,7 @@ Facendo il deployment di Kubernetes, ottieni un cluster.
...
 
-Questo è un diagramma di un cluster Kubernetes con tutti i componenti e le loro relazioni.
-
-![I componenti di Kubernetes](/images/docs/components-of-kubernetes.png)
-
-
+{{< figure src="/images/docs/components-of-kubernetes.svg" alt="I componenti di Kubernetes" caption="I componenti di un cluster Kubernetes" class="diagram-large" >}}
 
 <!-- body -->
 ## Componenti della Control Plane
```
Comment (sftim) [inline content/it/docs/concepts/overview/components.md:22]: Write “control plane” in lower case.

**Review by sftim — COMMENTED** [review]
Thanks. I do have some suggestions. Do they make sense?

**Comment by Fale** [comment]
Thanks @sftim ! I've implemented your suggestions :)

File: content/it/docs/concepts/overview/components.md [hunk 5]
```diff
@@ -46,12 +42,16 @@ Vedi [creare un cluster in High-Availability](/docs/admin/high-availability/) pe
...
 
-Alcuni esempi di controller gestiti dal kube-controller-manager sono:
+Ci sono molti tipi differenti di _controller_ (controllori). Alcuni esempi sono:
+
+  * Controllore dei Nodi: Responsabile del monitoraggio e della reazione quando i nodi del cluster diventano irraggiungibili.
+  * Controllore dei Job: Monitora gli oggetti di tipo Job, i quali rappresentano
+    delle operazioni una tantum, e crea i Pod necessari per la loro esecuzione.
+  * Controllore degli EndpointSlice: Popola gli oggetti EndpointSlice (che servono per
+    collegare i Service e i Pod).
+  * Controllore dei Service Account: Crea i ServiceAccount di default per i nuovi namespace.
```
Comment (fabriziopandini) [inline content/it/docs/concepts/overview/components.md:52]:
```suggestion
  * Node Controller: Responsabile del monitoraggio e della reazione quando i nodi del cluster diventano irraggiungibili.
  * Replication Controller: Monitora gli oggetti di tipo Job, i quali rappresentano
    delle operazioni una tantum, e crea i Pod necessari per la loro esecuzione.
  * Endpoints Controller: Popola gli oggetti EndpointSlice (che servono per
    collegare i Service e i Pod).
  * Service Account & Token Controllers: Crea i ServiceAccount di default per i nuovi namespace.
```
Eviterei traduzione letterale in questo caso, è una forzatura

File: content/it/docs/concepts/overview/components.md [hunk 6]
```diff
@@ -115,11 +115,17 @@ Il [Monitoraggio dei Container](/docs/tasks/debug-application-cluster/resource-u
...
 
+## {{% heading "whatsnext" %}}
+Scopri di più riguardo a:
+  * I [Nodi](/docs/concepts/architecture/nodes/) e [come comunicano](/docs/concepts/architecture/control-plane-node-communication/) con la  control plane.
+  * I Kubernetes [Controller](/docs/concepts/architecture/controller/).
+  * Il [kube-scheduler](/docs/concepts/scheduling/kube-scheduler/) che è
+  l'assegnatore di default di Kubernetes.
+  * La [documentazione](https://etcd.io/docs/) ufficiale di etcd.
+  * Molti dei [container runtime](/docs/setup/production-environment/container-runtimes/) di Kubernetes.
+  * Come integrare Kubernetes con i fornitori cloud usando il [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/).
```
Comment (fabriziopandini) [inline content/it/docs/concepts/overview/components.md:130]:
```suggestion
  * Come integrare Kubernetes con i cloud provider usando il [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/).
```

**Comment by Fale** [comment]
Thanks @fabriziopandini I've integrated the suggestions :)

File: content/it/docs/concepts/overview/components.md [hunk 7]
```diff
@@ -16,16 +16,12 @@ Facendo il deployment di Kubernetes, ottieni un cluster.
...
-Questo è un diagramma di un cluster Kubernetes con tutti i componenti e le loro relazioni.
-
-![I componenti di Kubernetes](/images/docs/components-of-kubernetes.png)
-
-
+{{< figure src="/images/docs/components-of-kubernetes.svg" alt="I componenti di Kubernetes" caption="I componenti di un cluster Kubernetes" class="diagram-large" >}}
 
 <!-- body -->
-## Componenti della Control Plane
+## Componenti della control plane
```
Comment (fabriziopandini) [inline content/it/docs/concepts/overview/components.md:22]: nit
```suggestion
## Componenti del control plane
```

File: content/it/docs/concepts/overview/components.md [hunk 8]
```diff
@@ -115,11 +115,17 @@ Il [Monitoraggio dei Container](/docs/tasks/debug-application-cluster/resource-u
...
-## {{% heading "whatsnext" %}}
-
-* Scopri i concetti relativi ai [Nodi](/docs/concepts/architecture/nodes/)
-* Scopri i concetti relativi ai [Controller](/docs/concepts/architecture/controller/)
-* Scopri i concetti relativi al [kube-scheduler](/docs/concepts/scheduling/kube-scheduler/)
-* Leggi la [documentazione](https://etcd.io/docs/) ufficiale di etcd
 
+## {{% heading "whatsnext" %}}
+Scopri di più riguardo a:
+  * I [Nodi](/docs/concepts/architecture/nodes/) e [come comunicano](/docs/concepts/architecture/control-plane-node-communication/) con la  control plane.
```
Comment (fabriziopandini) [inline content/it/docs/concepts/overview/components.md:124]:
```suggestion
  * I [Nodi](/docs/concepts/architecture/nodes/) e [come comunicano](/docs/concepts/architecture/control-plane-node-communication/) con il control plane.
```

**Comment by tengqm** [comment]
@Fale more feedbacks coming in, please help review and respond.

**Pull request closed without merging**